xml/ha_storage_protection.xml

```diff
@@ -809,10 +809,53 @@ Timeout (msgwait)  : 10
      </callout>
    </calloutlist>
   </step>
-  <step>
-   <para>For a two-node cluster, decide if you want predictable or random delays. For
+  <step xml:id="st-ha-storage-protect-fencing-static-random">
+   <para>For a <emphasis>two-node cluster</emphasis>, decide if you want to set the following delays. For
    other cluster setups you do not need to set this parameter.</para>
    <variablelist>
+    <varlistentry>
+     <term>Priority fencing delays</term>
+     <listitem>
+       <para>
+        This parameter delays nodes with the same priority (by default disabled).
+        Users set the priority of a resource which are more important to them.
+        A promoted resource has a higher priority than a started or a stopped one.
+        For a node, the priority is the sum of all priorities of resources running
+        on them.
```

Comment:
on them? It's unclear what _them_ refers to.

Reply:
Actually I made an error in the article. Them should be "it" and refers to the node. So all the "resources running on the node". I will rephrase the sentence.